java/core/src/test/org/apache/orc/impl/TestRecordReaderImpl.java

```diff
@@ -2449,4 +2449,41 @@ public void testSargApplier() throws Exception {
     f2.setAccessible(true);
     assertFalse((boolean)f2.get(applier1));
   }
+
+  @Test
+  public void testCompatibleSpecificationUnofficialFile() throws IOException {
+    // unofficial.orc is a file compatible with the ORC specification,
```

Comment:
What do you mean by `compatible`? Is it because ORC spec doesn't enforce `min` and `max`?

Reply:
> What do you mean by `compatible`? Is it because ORC spec doesn't enforce `min` and `max`?

Yes, the column statistics interface only needs to provide numberOfValues / hasNull / bytesOnDisk

Reply:
lets add the above info to the comment -- ideally with a link to the interface